src/DIRAC/Core/DISET/TransferClient.py

```diff
@@ -71,6 +71,7 @@ def sendFile(self, filename, fileId, token=""):
             retVal = transport.receiveData()
             return retVal
         finally:
+            fileHelper.oFile.close()
```

Comment:
oFile is not necessarily set

Reply:
I tried to follow down the list of calls and *seems* like it is set. Given the fact that the integration tests pass, we can be a bit confident that it is indeed set. But if you prefer, I put it in a `try...except` clause.

Reply:
It's only ever created in case the first argument of `getFileDescriptor` is a string, which is always true here, so you can forget my comment  !

Reply:
review OK then?